app/components/primer/alpha/select_panel.rb

```diff
@@ -397,6 +376,7 @@ def initialize(
         dynamic_label_prefix: nil,
         dynamic_aria_label_prefix: nil,
         body_id: nil,
+        use_experimental_remote_form: false,
```

Comment:
since these changes would actually solve `eventually_local` + form too, what do you think of calling this `use_experimental_non_local_form`? or would that be too confusing/overloaded? 🤔

Reply:
I'm on the fence 🤔 I don't want people to assume it will work for both until your changes are in. But It would be troublesome to change it after its merged.